ltext.py

```diff
@@ -533,10 +524,11 @@ def value(self):
         return self.text[self.start: self.end]
 
     def __repr__(self):
-        return 'Label({}, {}, value={})'.format(self.start, self.end, repr(self.value))
```

Comment:
I want `.value` to be seen rather than `.text`. The value of a label is what we care most, and much shorter than the whole text.